src/utils/reporting_workflow.js

```diff
@@ -131,11 +131,28 @@ function determineActionSequence(planDefinition) {
         // spec requires relationship SHALL === 'before-start'
         // so relatedAction is after currentAction
 
-        // directed edge from arg1 --> arg2
-        // TODO: track the offset on the edge label maybe?
-        g.setEdge(action.id, relatedAction.actionId);
-        // note that relatedAction.id doesn't already have to be in the graph
-        // it will be added if not
+        if (relatedAction.offsetDuration) {
+          // assume that the offsetDuration indicates a minimum, not a maximum.
+          // Duration object has a 'comparator' field which can be </<=/>/>=
+          // TODO: if there's a case of offset < x, just ignore it and run immediately
+          const { value, unit } = relatedAction.offsetDuration;
```

Comment:
In Duration it looks the code is used. I haven't worked with Duration before the example PlanDefinition has the code as "min" instead of unit as "min". The code system must be UCUM so we don't need to worry about different systems. http://hl7.org/fhir/R4/datatypes.html#Duration
```suggestion
          const { value, code } = relatedAction.offsetDuration;
```